SwiftzTests/StreamSpec.swift

```diff
@@ -149,5 +149,12 @@ class StreamSpec : XCTestCase {
 				}
 			}
 		}
-	}
+
+        property("sequence occurs in order") <- forAll { (xs : [String]) in
+            let seq = sequence(xs.map({ x in Stream.pure(x).take(1) }))
```

Comment:
It slows down to the point of not completing in any reasonable time. Assuming you mean wrapping it like other uses of `take`.
```
    return forAll { (n : UInt) in
    ...
    }
```